### ISSUE TITLE
New alerts added to default alert policy

### ISSUE THREAD

**Issue by jonwbstr** [MicrosoftDocs/microsoft-365-docs#479]
At least two new default alert policies were turned on within the last few days.
"Phish delivered due to tenant or user override" (previously temporary disabled)
"User impersonation phish delivered to inbox/folder"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9f2879ba-e58e-cd47-f488-10d153c31418
* Version Independent ID: 391a2c05-d63d-e84e-7c7d-80c3b3518419
* Content: [Alert policies in the security and compliance center](https://docs.microsoft.com/en-us/microsoft-365/compliance/alert-policies)
* Content Source: [microsoft-365/compliance/alert-policies.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/alert-policies.md)
* Service: **o365-seccomp**
* GitHub Login: @markjjo
* Microsoft Alias: **markjjo**

**Comment by mypil** [comment]
@jonwbstr - Thank you for submitting feedback.

I will get this issue over to the Microsoft 365 writing team for investigation.

Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.

**Comment by jonwbstr** [comment]
I've created several uservoice entries based on our experience with these two policies. Some of that feedback could potentially be addressed by linking from the alert configuration page to this article and adding more detailed information about how to manage the underlying policy generating these alerts.
 
e.g. "For more information about anti-phishing policies, see [Set up Office 365 ATP anti-phishing and anti-phishing policies](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/set-up-anti-phishing-policies#learn-about-atp-anti-phishing-policy-options)." 

https://office365.uservoice.com/forums/289138-office-365-security-compliance/suggestions/38665477-add-documentation-about-the-different-actions-that
https://office365.uservoice.com/forums/289138-office-365-security-compliance/suggestions/38665594-do-not-detect-messages-from-microsoft-as-phishing
https://office365.uservoice.com/forums/289138-office-365-security-compliance/suggestions/38665714-provide-additional-actions-to-help-us-take-special
https://office365.uservoice.com/forums/289138-office-365-security-compliance/suggestions/38665813-only-generate-one-notification-for-phishing-impers

**Comment by mypil** [comment]
@markjjo - Can you please share your insights on this issue?

Thank you.

cc: @kenwith

**Comment by markjjo** [comment]
@jonwbstr We can't repo; these two alert policies are not available in 3 PROD test tenants. So I have left the descriptions of them as-is (so that folks can create customer alert policies with the same functionality. As per you suggestion, I have added a link to documentation to ATP policies in the footnote. Thanks for the heads up.

**Comment by jonwbstr** [comment]
To close the loop on this, the policies were removed shortly after I reported their appearance. A week or two ago, they reappeared. I reported they were still causing undesirable behavior, and they got removed a second time. I spoke with several managers regarding the on, off, on, off again experience and was given assurances it would not happen a third time.

One policy is fine, two policies with overlapping functionality is not so great. Also, the past few weeks I have been working to reduce the false positives generated by the anti-phish policy.

Thanks,
-jon
